pages/documentation.css

```diff
@@ -3,22 +3,32 @@
 
 :root {
 	--gray: #e5e5e5;
+	--light-gray: #f5f5f5;
 	--white: #fff;
 	--black: #171717;
 	--blue: #1d4ed8;
 	--purple: #c13584;
 	--text-hover: #3b82f6;
-	--text-active: #262626;
 	--background-color: #fafafa;
 	--navigation-background: rgba(250, 250, 250, 0.5);
 }
 
 body {

```

Comment:
Can add the variables in a generic colors/variable file so that it is easy to get customised